src/open_inwoner/templates/pages/search.html

```diff
@@ -48,10 +48,29 @@ <h3 class="search-results__item-title">{{ hit.name }}</h3>
                     {% pagination page_obj=page_obj paginator=paginator request=request %}
                 </div>
             </div>
+        {% else %}
+            <div class="grid">
+                {% spaceless %}
+                <div class="grid__sidebar">&nbsp;</div>
+                {% endspaceless %}
```

Comment:
Does this/theses `spaceless` tags around a single tag actually do anything? I see similar on line 75 but it looks strange, like there would still be space before/after the template tags?

Reply:
@Bartvaderkin Ah yes, I thought it was a styling tag, but it's not. Spaceless tag is a standard Django template tag, used because newlines/spaces in templates behave like a character. Only whitespaces between successive opening HTML tags and successive closing tags are removed, so not effective for single elements.